Jenkinsfile

```diff
@@ -5,23 +5,27 @@ node {
     def workspace = pwd()
     //Always wrap each test block in a timeout
     //This first block sets up engine within 15 minutes
-    timeout(15) {
-      stage('Build Grakn') {//Stages allow you to organise and group things within Jenkins
-        sh 'npm config set registry http://registry.npmjs.org/'
-        checkout scm
-        sh 'if [ -d maven ] ;  then rm -rf maven ; fi'
-        sh "mvn versions:set -DnewVersion=${env.BRANCH_NAME} -DgenerateBackupPoms=false"
-        sh 'mvn clean package -DskipTests -U -Djetty.log.level=WARNING -Djetty.log.appender=STDOUT'
-        archiveArtifacts artifacts: "grakn-dist/target/grakn-dist*.tar.gz"
-      }
-      stage('Init Grakn') {
-        sh 'if [ -d grakn-package ] ;  then rm -rf grakn-package ; fi'
-        sh 'mkdir grakn-package'
-        sh 'tar -xf grakn-dist/target/grakn-dist*.tar.gz --strip=1 -C grakn-package'
-        sh 'grakn-package/bin/grakn.sh start'
-      }
-      stage('Test Connection') {
-        sh 'grakn-package/bin/graql.sh -e "match \\\$x;"' //Sanity check query. I.e. is everything working?
+    withEnv([
```

Comment:
that's right, I didn't know whether I could nest withEnv calls so have left it for a general clean up at the end. I know there are too may hard coded paths, but again I am hoping to clean them up at the end